lib/install.js

```diff
@@ -7,7 +7,6 @@ var debug = require('debug')('strong-pm:install');
 var fs = require('fs');
 var path = require('path');
 var slServiceInstall = require('strong-service-install');
-
```

Comment:
I prefer this blank line be left here, if this file even needs to be modified (see other comment about removing `--json-file-db` from `sl-pm-install`).